packages/terminal/src/browser/terminal-widget.ts

```diff
@@ -95,9 +95,10 @@ export class TerminalWidget extends BaseWidget {
 
     public async start(): Promise<void> {
         this.registerResize();
-        const root = await this.workspaceService.root;
+        const root = await this.workspaceService.tryRoot;
+        const rootUri = root !== undefined ? root.uri : undefined;
         this.terminalId = await this.shellTerminalServer.create(
-            { rootURI: root.uri, cols: this.cols, rows: this.rows });
+            { 'rootURI': rootUri, cols: this.cols, rows: this.rows });
```

Comment:
Why did you have to change `rootURI` to `'rootURI'`?

Reply:
@hexa00 you can rename `rootUri` variable to `rootURI` and use structuring: `{ rootURI, cols: this.cols, rows: this.rows }`

Reply:
I thought I needed to distinguish between the key and it's value since they have the same name, but that's not necessary indeed and @akosyakov version is even better. Thx! Fixing with this.